tests/commands/create/test_install_app_support_package.py

```diff
@@ -323,9 +268,6 @@ def test_install_pinned_custom_app_support_package_url_with_args(
 ):
     """A custom support package can be specified as URL with args, and pinned
     to a revision."""
```

Comment:
If I understand correctly, it can no longer be pinned to a revision: the URL is just used as-is. In which case, this whole test function isn't really necessary anymore: it was only added because we knew we were modifying the URL.

Reply:
It *can* be pinned, as long as you're not specifying a custom template as well. The use case here is someone actively opting into a new version of an official support package, or explicitly *not* opting into a new version of a support package - e.g., if we published an update to the 3.10.0 branch to use a new support revision, a user could pin the older support revision.

Reply:
I've added some extra tests around this (and found a bug in the process - around explicit branches in the development version).

Reply:
Looking closer, the problem here is that the test name and description is misleading. Testing a support URL that *has* args is still valid (although it's less of an issue than before, because weren't actually manipulating those args).

I've also added a warning (plus tests) for the user if they specify a custom template and a template revision.